src/documentation/sdks/python-custom-io.md

```diff
@@ -45,7 +45,7 @@ You can use test harnesses and utility methods available in the [source_test_uti
 
 ## <a name="creating-sources"></a>Creating a Custom Source
```

Comment:
There are several more `custom` words on this page -- perhaps clean up all of them?